README.md

```diff
@@ -64,30 +64,7 @@ The following Kubernetes versions are supported as we tested against these versi
 ## 🚀 QuickStart
 
 ### Install OpenFunction
-
-Visit [ofn releases page](https://github.com/OpenFunction/cli/releases) to download the latest version of `ofn`, the CLI of OpenFunction, to install OpenFunction and its dependencies on your Kubernetes cluster.
-
-Besides, you can perform the following steps to install the latest version of OpenFunction.
-
-1. Run the following command to download `ofn`.
-
-   ```
-   wget -c  https://github.com/OpenFunction/cli/releases/latest/download/ofn_linux_amd64.tar.gz -O - | tar -xz
-   ```
-
-2. Run the following commands to make `ofn` executable and move it to `/usr/local/bin/`.
-
-   ```
-   chmod +x ofn && mv ofn /usr/local/bin/
-   ```
-
-3. Run the following command to install OpenFunction.
-
-   ```
-   ofn install --all
-   ```
-
-You can refer to [ofn install document](https://github.com/OpenFunction/cli/blob/main/docs/install.md) for more information about the `ofn install` command.
+See our [Installation](https://openfunction.dev/docs/getting-started/installation/#install-openfunction) guide over in our docs.
```

Comment:
=>
To install OpenFunction, please refer to [Installation Guide](https://openfunction.dev/docs/getting-started/installation/#install-openfunction)